plugins/ChangeAwaitImport.scala

```diff
@@ -3,27 +3,21 @@ import scala.meta._
 
 class ChangeAwaitImport extends SyntacticRule("ChangeAwaitImport") {
 
+  def findAwaitImportee(importees: List[scala.meta.Importee]): Option[Importee] = {
+    importees.find{ _ match {
+      case i @ Importee.Name(Name("Await")) => true
```

Comment:
```suggestion
      case importee"Await" => true
```

if you like semiquotes :P